src/System.Collections/tests/BitArray/BitArray_CtorTests.cs

```diff
@@ -236,6 +236,16 @@ public static void Ctor_Simple_Method_Tests()
             Assert.False(bitArray.IsReadOnly);
             Assert.Equal(bitArray, bitArray.Clone());
         }
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "A bug in BitArray.Clone() caused an ArgumentExeption to be thrown in this case.")]
+        [Fact]
+        public static void Clone_LargeLength_Works()
```

Comment:
super nit: `LargeLength` is weird. It should either be `LongLength` or `LargeSize`.